### ISSUE TITLE
Remove timestamp from loglines ?

### ISSUE THREAD

**Issue by benrubson** [vgough/encfs#270]
Hello,

Log lines now come with a timestamp :
```
Jan 23 11:47:51 srv1 encfs: 2017-01-23 11:47:51,372 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:48:01 srv1 encfs: 2017-01-23 11:48:01,761 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:48:12 srv1 encfs: 2017-01-23 11:48:12,028 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:48:22 srv1 encfs: 2017-01-23 11:48:22,384 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:48:32 srv1 encfs: 2017-01-23 11:48:32,740 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:48:43 srv1 encfs: 2017-01-23 11:48:43,014 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:48:53 srv1 encfs: 2017-01-23 11:48:53,259 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:49:03 srv1 encfs: 2017-01-23 11:49:03,573 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:49:14 srv1 encfs: 2017-01-23 11:49:14,025 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
Jan 23 11:49:24 srv1 encfs: 2017-01-23 11:49:24,329 ERROR [default] Filesystem /home/encfs/ inactivity detected, but failed to unmount
```

Syslog is then unable to merge them like this :
`Jan 23 11:50:29 srv1 last message repeated 17 times`

Could we then think about removing the timestamp field from log messages ?
Does easylogging allow this ?

Many thanks 👍 

Ben

**Comment by benrubson** [comment]
Solved by #271 !

**Comment by benrubson** [comment]
#296 merged !
